src/ggcmpy/openggcm.py

```diff
@@ -186,6 +188,11 @@ def __init__(self, xarray_obj: Any):
     def __repr__(self) -> str:
         return "OpenGGCM accessor\n" + repr(self._coords)
 
+    def plot(
+        self, file: str, var: str, lats_max: int, lats_min: int, spacing: int, mlt: bool
+    ) -> Any:
+        return pp.plot(file, var, lats_max, lats_min, spacing, mlt)
+
```

Comment:
The accessor should get called on an already available `DataArray`. There should be no need to pass `file` or `var`.

For the other args, I think it should make sense to have them be optional, and actually just use `**kwargs` so you don't have to duplicate the logic of what arguments may be passed here.